console/views.py

```diff
@@ -1,8 +1,57 @@
 import logging
+import requests
 
+from flask import render_template
+from flask import request
 from structlog import wrap_logger
 
-import console.settings as settings
+from console import app
+from console import settings
+from console.helpers.exceptions import ClientError, ServiceError
 
 logging.basicConfig(level=settings.LOGGING_LEVEL, format=settings.LOGGING_FORMAT)
 logger = wrap_logger(logging.getLogger(__name__))
+
+
+def send_data(url, data):
+    try:
+        r = requests.post(url, data)
+    except requests.exceptions.ConnectionError as e:
+        logger.error('Could not connect to sdx-decrypt', response="connection error")
+        raise e
+        return
+
+    if (r.status_code > 199 and r.status_code < 300):
```

Comment:
The parentheses aren't need in the if and elif statements.

It could also be simplified to:
if 199 < r.status_code < 300: